index.html

```diff
@@ -32,16 +32,16 @@
       <main>
         <section class="headline">
           <div>
-            <h2 class="headline-text">
+            <h1 class="headline-text">
```

Comment:
- [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1.
![image](https://user-images.githubusercontent.com/62290545/187517749-18372c47-530c-4eec-9e2c-e99e36112e29.png)
![image](https://user-images.githubusercontent.com/62290545/187517899-e542d248-1d41-4931-a537-e3075a4e4035.png)
![image](https://user-images.githubusercontent.com/62290545/187517988-98aa5289-fe42-4d75-8af9-c8d3fc8d74de.png)

- You can use [this tool](https://www.aditus.io/button-contrast-checker/) to check on the color contrast ratio.